src/Our.Umbraco.HeadRest/Composing/HeadRestComponent.cs

```diff
@@ -1,17 +0,0 @@
-﻿using System.Web.Routing;
-using Umbraco.Core.Composing;
-using Umbraco.Web;
-
-namespace Our.Umbraco.HeadRest.Composing
-{
-    public class HeadRestComponent : IComponent
-    {
-        public void Initialize()
-        {
-            RouteTable.Routes.IgnoreStandardExclusions();
```

Comment:
I couldn't find a netcore equivalent to this and the package seems to work in v11 without it - not sure if it's needed in netcore?